blocks/shared-styles/_children/promo-date/index.scss

```diff
@@ -1,11 +1,11 @@
 .promo-date {
```

Comment:
promo-date is now the date component, so this should not be used anymore and can probably be deleted